scripts/exec

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+from github import Github
+
+# Get inputs from shell
+(token, organization_name, path) = sys.argv[1:4]
+
+# Fetch organization from bot repo, then get all organization repos
+organization = Github(token).get_organization(organization_name)
+org_repos = organization.get_repos()
+
+for repo in org_repos:
+    print("Organizing " + repo.name)
+
+    # Automatically merge PRs that meet criteria
+    prs = repo.get_pulls()
+    for pr in prs:
+        pr_reviews = pr.get_reviews()
+        
+        # Parse reviews for merge criteria.
+        changes_requested = False
+        approving_reviews = 0
+        for review in pr_reviews:
+            if review.state == "APPROVED":
```

Comment:
What other states are there? There could be subtle bugs here...

Reply:
There is surprisingly no documentation on the `state` field for `PullRequestReview` on Github's end: https://developer.github.com/v3/pulls/reviews/

They do provide examples in the sample responses, but no definitive listing of all possible states. The ones I can see are:

- APPROVED
- CHANGES_REQUESTED
- PENDING
- DISMISSED

The last one might cause some problems with our criteria that all "Changes Requested" reviews be updated to "Approved", though I don't know exactly how that state works.